src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -162,17 +162,17 @@ void pychip_DeviceController_SetLogFilter(uint8_t category)
 }
 
 CHIP_ERROR pychip_DeviceController_ConnectBLE(chip::Controller::DeviceCommissioner * devCtrl, uint16_t discriminator,
-                                              uint32_t setupPINCode)
+                                              uint32_t setupPINCode, chip::NodeId nodeid)
 {
-    return devCtrl->PairDevice(kRemoteDeviceId,
+    return devCtrl->PairDevice(nodeid,
```

Comment:
Node ID cannot be expected to be known at commissioning/rendezvous time. Is this a permanent change?

Reply:
this is temp change.